src/views/start.html

```diff
@@ -1,11 +1,31 @@
 {% extends "layout.html" %}
 
 {% block content %}
-    <div class="govuk-grid-row">
+
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper">
+      <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">GitHub Requests</h1>
+            <h1 class="govuk-heading-l">GitHub Requests</h1>
+
+            <p class="govuk-body">The GitHub request application is a tool designed to streamline and automate the process of managing and tracking requests for GitHub. This includes the adding, removal or editing of members. Further documentation can be found <a href="https://github.com/cabinetoffice/github-requests-app" class="govuk-link">here</a>.</p>
+
+            <h1 class="govuk-heading-m">Before you start:</h1>
 
-            <p class="govuk-body-l">Use this service to make member, team, and repository related requests to the cabinetoffice GitHub organisation.</p>
+            <ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+                <li>
+                    <p class="govuk-body">New users must have a GitHub handle.</p>
+                </li>
+                <li>
+                    <p class="govuk-body">New user has to be associated with an existing team; otherwise, the new team must be created in the same request.</p>
+                </li>
+                <li>
+                    <p class="govuk-body">Every new team has to have at least one maintainer (<a href="https://docs.github.com/en/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member#about-team-maintainers" class="govuk-link">Team Maintainer Docs</a>).</p>
+                </li>
+                <li>
+                    <p class="govuk-body">Non permanent users/members must provide the expected end date of the work.</p>
```

Comment:
Non-perms end of contract makes more sense to me. To maintain consistency with other points - "New users that are not permanent staff must provide an expected end date of their contract"